composables/paginator.ts

```diff
@@ -35,6 +35,13 @@ export function usePaginator<T, P, U = T>(
     data.splice(0, 0, status)
   }
 
+  function deleteItem(id: string) {
+    const data = items.value as mastodon.v1.Status[]
```

Comment:
took these lines of code straight from the "delete" event handler below in this file